src/tests/test_plot.py

```diff
@@ -113,18 +113,19 @@ def test_summary_stats_missing_file(chinook_db, ip_empty):
 
 
 @pytest.mark.parametrize(
-    "cell, error_type, error_message",
+    "cell, error_",
     [
         [
             "%sqlplot histogram --table data.csv --column age --table data.csv",
-            ValueError,
-            "Data contains NULLs",
+            None,
         ]
     ],
 )
 # Test internal plot function e.g.
-def test_internal_histogram_exception(tmp_empty, ip, cell, error_type, error_message):
-    Path("data.csv").write_text("name,age\nDan,33\nBob,19\nSheri,")
+def test_internal_histogram_null_support(tmp_empty, ip, cell, error_):
```

Comment:
Add another test-case for the success case (data doesn't contain nulls)

Reply:
Added a test case so where data doesn't have any nulls